### PR TITLE
Adds cross-origin prefetch article

### PR DESCRIPTION
**Pull request by rachelandrew**
To merge with other two articles, I/O content.

### CODE REVIEW

File: site/en/blog/cross-origin-prefetch/index.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Speeding up Largest Contentful Paint
+
+As the saying goes, you can't improve what you don't measure. For user experiences on the web, we believe that [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) constitute a solid set of user-centric metrics designed to capture fundamental aspects of the user experience. In particular, [Largest Contentful Paint](https://web.dev/lcp/), or LCP, measures your pages' loading performance by reporting the time it takes to display the largest text or image block that the user sees. To provide a good user experience, LCP should occur within 2.5 seconds of when the page first starts loading (namely, the good LCP threshold).
```
Comment (malchata) [inline site/en/blog/cross-origin-prefetch/index.md:27]:
```suggestion
As the saying goes, you can't improve what you don't measure. For user experiences on the web, we believe that [Core Web Vitals](https://web.dev/vitals/#core-web-vitals) constitute a solid set of user-centric metrics designed to capture fundamental aspects of the user experience. In particular, [Largest Contentful Paint (LCP)](https://web.dev/lcp/) measures your pages' loading performance by reporting the time it takes to display the largest text or image block that the user sees. To provide a good user experience, LCP should occur within 2.5 seconds of when the page first starts loading (namely, the good LCP threshold).
```

File: site/en/blog/cross-origin-prefetch/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Let's look at what contributes to the LCP of a typical page. 
+
+<figure>
+  {% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/v5Fl8GPalSO3brhBWGds.png", alt="Page loading waterfall.", width="800", height="450" %}
+
+  <figcaption>A typical waterfall required to load a web page</figcaption>
```
Comment (malchata) [inline site/en/blog/cross-origin-prefetch/index.md:34]:
```suggestion
  {% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/v5Fl8GPalSO3brhBWGds.png", alt="Page loading waterfall.", width="800", height="450" %}
  <figcaption>A typical waterfall required to load a web page</figcaption>
```

File: site/en/blog/cross-origin-prefetch/index.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+</figure>
+
+When a user visits a page, the browser requests the HTML from the server. The server responds with the HTML, which gives the browser more hints on what to fetch next, including CSS, JavaScript, fonts, and images. As these responses come back, the browser must also do some work to evaluate them, and eventually lay out and paint components on the page. But the majority of time is spent waiting for those packets to travel from the device to the server and then back to the device. In fact, our data (Chrome for Android; median) show that about 40% of user-visible latency is typically spent by the browser waiting for the very first byte to come back from the server.
```
Comment (malchata) [inline site/en/blog/cross-origin-prefetch/index.md:37]:
```suggestion
When a user visits a page, the browser requests the HTML from the server. The server responds with the HTML, which gives the browser more hints on what to fetch next, including CSS, JavaScript, fonts, and images. As these responses come back, the browser must also do some work to evaluate them, and eventually lay out and paint components on the page. But the majority of time is spent waiting for those packets to travel from the device to the server and then back to the device. In fact, our data (Chrome for Android; median) show that about 40% of user-visible latency is typically spent by the browser waiting for [the very first byte](https://web.dev/ttfb/) to come back from the server.
```

File: site/en/blog/cross-origin-prefetch/index.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+## The power of prefetching
+
+If one could prefetch all these files—that is, fetch them before the user visits the page, then this would provide a massive speed boost, only leaving a few tasks before displaying the page: evaluating, calculating the layout, and painting.
+
+<figure>
+
+  {% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/9UOE4fsPHr7SYLkmO2zZ.png", alt="Streamlined waterfall.", width="800", height="450" %}
+
+  <figcaption>With all resources preloaded, the waterfall is perfectly streamlined.</figcaption>
+</figure>
```
Comment (malchata) [inline site/en/blog/cross-origin-prefetch/index.md:48]:
```suggestion
<figure>
  {% Img src="image/kheDArv5csY6rvQUJDbWRscckLr1/9UOE4fsPHr7SYLkmO2zZ.png", alt="Streamlined waterfall.", width="800", height="450" %}
  <figcaption>With all resources preloaded, the waterfall is perfectly streamlined.</figcaption>
</figure>
```

File: site/en/blog/cross-origin-prefetch/index.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Given the data shared earlier, one could also just prefetch the main resource and still achieve a significantly faster page load. In the same-site case, this type of technique is readily available with primitives such as [link rel=prefetch](https://web.dev/link-prefetch/). However, with cross-site scenarios, it's not as straightforward.
```
Comment (malchata) [inline site/en/blog/cross-origin-prefetch/index.md:50]:
```suggestion
Given the data shared earlier, one could also just prefetch the main resource and still achieve a significantly faster page load. In the same-site case, this type of technique is readily available with primitives such as [`rel=prefetch`](https://web.dev/link-prefetch/). However, with cross-site scenarios, it's not as straightforward.
```

File: site/en/blog/cross-origin-prefetch/index.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+Suppose that a referrer site were to instruct the browser to prefetch a page from a different site. Obviously, when the user clicks on the link to this pre-fetched page, they would enjoy a better user experience with a much faster page load. However, what if the user never clicks on this link? They wouldn't expect a linked website to learn that they might have been interested in a topic while they were browsing for it on the referrer site. And, yet this is a substantial risk because the prefetching requests would carry the user's IP address, and cookies if any, as any other regular request.
+
+## Solutions
+
+To enable privacy-safe cross-site prefetching, we've developed two solutions over the last 3 years: [private prefetch proxy](/blog/private-prefetch-proxy) and [signed exchanges](http://goo.gle/sxg-info) (SXG). We also ran a large-scale experiment to confirm that cross-origin prefetch delivers significant speed benefits. Concretely, when we looked at instances where Google was able to safely prefetch the main HTML for the user's next navigation, we saw that the fraction of page loads with Good LCP increased by 14 percentage points!
```
Comment (malchata) [inline site/en/blog/cross-origin-prefetch/index.md:60]:
```suggestion
To enable privacy-safe cross-site prefetching, we've developed two solutions over the last 3 years: [Private Prefetch Proxy](/blog/private-prefetch-proxy) and [Signed Exchanges (SXG)](http://goo.gle/sxg-info). We also ran a large-scale experiment to confirm that cross-origin prefetch delivers significant speed benefits. Concretely, when we looked at instances where Google was able to safely prefetch the main HTML for the user's next navigation, we saw that the fraction of page loads with "good" LCP increased by 14 percentage points!
```

File: site/en/blog/cross-origin-prefetch/index.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    -  If you currently encode personalization directly into your HTML, you can still continue to do so when cookies are present, and use lazy-loading as a fallback strategy for prefetched pages.
+    -  If your site is not personalized based on cookies, or if the personalization isn't critical, you can choose to serve the same content to your repeat visitors as you would to first-time visitors.
+
+At the moment, Private Prefetch Proxy is only enabled for first-time visitors (links without cookies) with on-going work to expand coverage to repeat visitors (links with cookies).  On the other hand, Signed Exchange already supports cross-site prefetching for both first-time and repeat visitors (with the approaches outlined above).
```
Comment (malchata) [inline site/en/blog/cross-origin-prefetch/index.md:75]:
```suggestion
At the moment, Private Prefetch Proxy is only enabled for first-time visitors (links without cookies) with on-going work to expand coverage to repeat visitors (links with cookies). On the other hand, Signed Exchange already supports cross-site prefetching for both first-time and repeat visitors (with the approaches outlined above).
```

**Review by malchata — APPROVED** [review]
Aside from some nits, LGTM!

**Comment by KenjiBaheux** [comment]
Can you also adjust my author byline to match Michael's?

"Kenji is a product manager on the Chrome team."

File: site/en/blog/cross-origin-prefetch/index.md [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+
+### Extra data serving from prefetch
+
+Enabling cross-site prefetching can result in extra data serving. Indeed, if a referrer prefetches your page but the user doesn't click on the link, this would represent additional traffic for you.
+
+-  To mitigate this, one could request that the referrer be less aggressive with their prefetch requests. Similarly, the referrer, or the browser, can mitigate this by focusing on relatively lightweight, yet critical, resources (for example, main resource, critical CSS, or JavaScript sub-resources). This is essentially a trade-off between speed benefits and extra traffic.
+-  Alternatively, one could offset this traffic by opting into additional caching (see the [deep dive on Signed Exchange for more details](http://goo.gle/sxg-info)). The downside here would be that caching content for too long can result in showing older information to your users. This is essentially a trade-off between extra data serving and content freshness.
```
Comment (KenjiBaheux) [inline site/en/blog/cross-origin-prefetch/index.md:79]: Can you replace the text and link on this paragraph as follows:

```
Alternatively, one could offset this traffic by opting into additional caching (see [this section on Signed Exchange](https://web.dev/signed-exchanges/#:~:text=The%20expires%20parameter%20in%20the%20signature%20indicates%20a%20SXG%27s%20expiration%20date.%20A%20SXG%20may%20be%20valid%20for%20at%20most%207%20days.%20Find%20more%20information%20on%20the%20signature%20header%20in%20the%20Signed%20HTTP%20Exchanges%20spec)  for more details).
```

**Comment by KenjiBaheux** [comment]
With these changes in, I think the article is good to go. Timing wise, it should be published with the other 2 articles, before the IO session on day 2 at 9am pdt.

File: site/en/blog/cross-origin-prefetch/index.md (hunk 8 above)
Comment (rachelandrew) [inline site/en/blog/cross-origin-prefetch/index.md:79]:
```suggestion
-  Alternatively, one could offset this traffic by opting into additional caching (see [this section on Signed Exchange](https://web.dev/signed-exchanges/#:~:text=The%20expires%20parameter%20in%20the%20signature%20indicates%20a%20SXG%27s%20expiration%20date.%20A%20SXG%20may%20be%20valid%20for%20at%20most%207%20days.%20Find%20more%20information%20on%20the%20signature%20header%20in%20the%20Signed%20HTTP%20Exchanges%20spec)  for more details).
```

Comment (rachelandrew) [inline site/en/blog/cross-origin-prefetch/index.md:79]:
```suggestion
-  Alternatively, one could offset this traffic by opting into additional caching (see [this section on Signed Exchange](https://web.dev/signed-exchanges/#:~:text=The%20expires%20parameter%20in%20the%20signature%20indicates%20a%20SXG%27s%20expiration%20date.%20A%20SXG%20may%20be%20valid%20for%20at%20most%207%20days.%20Find%20more%20information%20on%20the%20signature%20header%20in%20the%20Signed%20HTTP%20Exchanges%20spec)  for more details). The downside here would be that caching content for too long can result in showing older information to your users. This is essentially a trade-off between extra data serving and content freshness.
```

File: site/_data/i18n/authors.yaml [hunk 9]
```diff
@@ -795,3 +795,14 @@ rkochman:
     en: 'Rob Kochman'
   description:
     en: "Rob is a product manager on Chrome's Web Platform team."
+kenjibaheux:
+  title:
+    en: Kenji Baheux
+  description:
+    en: Kenji is a product manager at Chrome.
```
Comment (rachelandrew) [inline site/_data/i18n/authors.yaml:802]:
```suggestion
    en: Kenji is a product manager on the Chrome team.
```

**Comment by rachelandrew** [comment]
@KenjiBaheux changes are in, as mentioned on the other post. Let me know if you want me to publish these today (11th) or tomorrow.

**Comment by KenjiBaheux** [comment]
Today or tomorrow, as long as it's live before the talk :)

**Pull request merged**